en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md

```diff
@@ -0,0 +1,286 @@
+               <msazurestorage.listBlobs>
+                   <containerName>{$ctx:containerName}</containerName>
+               </msazurestorage.listBlobs>
+               <log level="full">
+                   <property name="List uploaded emplyee details" value="List uploaded emplyee details"/>
+               </log>
+               <respond/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+       <resource methods="POST" url-mapping="/deletedetails">
+           <inSequence>
+               <property expression="json-eval($.accountName)" name="accountName" scope="default" type="STRING"/>
+               <property expression="json-eval($.accountKey)" name="accountKey" scope="default" type="STRING"/>
+               <property expression="json-eval($.containerName)" name="containerName" scope="default" type="STRING"/>
+               <property expression="json-eval($.fileName)" name="fileName" scope="default" type="STRING"/>
+               <msazurestorage.init>
+                   <accountName>eiconnectortest</accountName>
+                   <accountKey>bWt69gFpheoD6lwVsMgeV5io2/KxlXK1KUcod68PhzuV1xHxje0LBD4Bd+y+ESAOlH5BTAfvdDG5q4Hhg==</accountKey>
+               </msazurestorage.init>
+               <msazurestorage.deleteBlob>
+                   <containerName>{$ctx:containerName}</containerName>
+                   <fileName>{$ctx:fileName}</fileName>
+               </msazurestorage.deleteBlob>
+               <log level="full">
+                   <property name="Delete selected employee details" value="Delete selected employee details"/>
+               </log>
+               <respond/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+       <resource methods="POST" url-mapping="/deletecontainer">
+           <inSequence>
+               <property expression="json-eval($.accountName)" name="accountName" scope="default" type="STRING"/>
+               <property expression="json-eval($.accountKey)" name="accountKey" scope="default" type="STRING"/>
+               <property expression="json-eval($.containerName)" name="containerName" scope="default" type="STRING"/>
+               <msazurestorage.init>
+                   <accountName>eiconnectortest</accountName>
+                   <accountKey>bWt69gFpheoD6lwVsMgeV5io2/KxlXK1KUcod68PhzuV1xHxje0LBD4Bd+y+ESAOlH5BTAfvdDG5q4Hhg==</accountKey>
+               </msazurestorage.init>
+               <msazurestorage.deleteContainer>
+                   <containerName>{$ctx:containerName}</containerName>
+               </msazurestorage.deleteContainer>
+               <log level="full">
+                   <property name="Delete selected container" value="Delete selected container"/>
+               </log>
+               <respond/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+   </api>
+   ```
+**Note**
+
```

Comment:
Please modify below properties of the configuration  as applicable.

Reply:
Updated.